src/main/scala/com/bazaarvoice/sswf/SSWFStep.scala

```diff
@@ -0,0 +1,19 @@
+package com.bazaarvoice.sswf
+
+/**
+ * Your Step Enum must implement/extend this
+ */
+trait SSWFStep {
+  /** The name of the Step */
+  def name: String
+  /** The version of the step definition (increment if you change any options) */
```

Comment:
This is not the version you're thinking of ;), which I guess just underscores the need for the doc you're asking for. I think "patterns" is a good idea.